sympy/integrals/manualintegrate.py

```diff
@@ -436,14 +440,17 @@ def make_second_step(steps, integrand):
                              make_second_step(steps[1:], v * du),
                              integrand, symbol)
         else:
-            return integral_steps(integrand, symbol)
+            if integral_steps(integrand, symbol):
+                return integral_steps(integrand, symbol)
```

Comment:
There should be only one call of `integral_steps`. Its return value can be saved and only then tested.